plugins/quickbooks-online/src/src/Facade/QuickBooksFacade.php

```diff
@@ -471,8 +471,8 @@ function (array $a, array $b) {
                 );
             }
 	    }
-            $pluginData->lastExportedPaymentID = $ucrmPayment['id'];
-            $this->optionsManager->update();
+        $pluginData->lastExportedPaymentID = $ucrmPayment['id'];
+        $this->optionsManager->update();
```

Comment:
Hi @raymondjclark, the change is whitespace only, did you create the PR correctly?

Reply:
Yes, these two lines, indented were assumed part of the non-credit payment code. I believe this change will have them execute if a payment only contains a credit.

Reply:
Whitespace has absolutely no effect on code execution, it could all be on single line and still behave the same. :)

Reply:
Gotcha. Here is the problem, when a payment only contains a credit those lines do not get executed and they need to.